iocore/eventsystem/I_VConnection.h

```diff
@@ -32,6 +32,7 @@
 #error "include I_VIO.h"
 #endif
 
+#define TS_VCONN_MAX_USER_ARG 16
```

Comment:
Let's drop this down to 4 or 8 - I doubt we'll need even that many and there are a *lot* of these in a live system.